lib/models/updateAction.js

```diff
@@ -92,6 +92,9 @@ function buildUpdateOptions(action, event) {
         id: myutils.expandVar(action.parameters.id, event),
         isPattern: myutils.expandVar(action.parameters.isPattern, event)
     };
+    if (myutils.expandVar(action.parameters.updateType, event)) {
+        options.updateType = myutils.expandVar(action.parameters.updateType, event);
```

Comment:
Maybe it should be better to use "actionType" literal instead of "updateType", as the former is the same term used in NGSI documentation.